CMakeLists.txt

```diff
@@ -72,6 +74,8 @@ set(MODULES_HEADERS
   ${CMAKE_SOURCE_DIR}/src/Helpers.hpp
   ${CMAKE_SOURCE_DIR}/src/Logger.hpp
   ${CMAKE_SOURCE_DIR}/src/ModulesFactory.hpp
+  ${CMAKE_SOURCE_DIR}/src/CommandResourceEstimator.hpp
```

Comment:
Please provide a small documentation header in hpp files to quickly describe what are those methods supposed to do, what info they expect when running, ... You can get inspiration for the format if you look at Hook and Isolator headers.

Reply:
thanks for the feeback, i will make it more understandable!